### ISSUE TITLE
Failure to install with cargo

### ISSUE THREAD

**Issue by wbthomason** [mtshiba/pylyzer#52]
Thanks for the interesting project! Unfortunately, I'm now running into issues installing `pylyzer` via `cargo`:
```
❯ cargo install pylyzer
    Updating crates.io index
  Installing pylyzer v0.0.43
    Updating crates.io index
   Compiling libc v0.2.147
   Compiling proc-macro2 v1.0.66
   Compiling unicode-ident v1.0.11
   Compiling autocfg v1.1.0
   Compiling cfg-if v1.0.0
   Compiling typenum v1.16.0
   Compiling version_check v0.9.4
   Compiling syn v1.0.109
   Compiling parking_lot_core v0.9.8
   Compiling getrandom v0.1.16
   Compiling ryu v1.0.15
   Compiling block-padding v0.2.1
   Compiling erg_common v0.6.20
   Compiling generic-array v0.14.7
   Compiling lock_api v0.4.10
   Compiling either v1.9.0
   Compiling ppv-lite86 v0.2.17
   Compiling once_cell v1.18.0
   Compiling opaque-debug v0.3.0
   Compiling keccak v0.1.4
   Compiling itertools v0.9.0
   Compiling quote v1.0.33
   Compiling rand_core v0.5.1
   Compiling syn v2.0.31
   Compiling smallvec v1.11.0
   Compiling base64 v0.13.1
   Compiling rand_chacha v0.2.2
   Compiling scopeguard v1.2.0
   Compiling rand v0.7.3
   Compiling num-traits v0.2.16
   Compiling tinyvec_macros v0.1.1
   Compiling block-buffer v0.9.0
   Compiling digest v0.9.0
   Compiling siphasher v0.3.11
   Compiling rand_core v0.6.4
   Compiling sha3 v0.9.1
   Compiling crunchy v0.2.2
   Compiling serde v1.0.188
   Compiling thread_local v1.1.7
   Compiling rand v0.8.5
   Compiling phf_shared v0.11.2
   Compiling malachite-base v0.3.2
   Compiling tinyvec v1.6.0
   Compiling parking_lot v0.12.1
   Compiling num-integer v0.1.45
   Compiling anyhow v1.0.75
   Compiling paste v1.0.14
   Compiling tiny-keccak v2.0.2
   Compiling phf_generator v0.11.2
   Compiling unicode-normalization v0.1.22
   Compiling unicode-bidi v0.3.13
   Compiling serde_derive v1.0.188
   Compiling unic-char-range v0.9.0
   Compiling convert_case v0.4.0
   Compiling unic-common v0.9.0
   Compiling percent-encoding v2.3.0
   Compiling pmutil v0.5.3
   Compiling serde_json v1.0.105
   Compiling memchr v2.6.3
   Compiling Inflector v0.11.4
   Compiling form_urlencoded v1.2.0
   Compiling erg_compiler v0.6.20
   Compiling unic-ucd-version v0.9.0
   Compiling unic-char-property v0.9.0
   Compiling rustpython-parser-vendored v0.3.0
   Compiling idna v0.4.0
   Compiling phf_codegen v0.11.2
   Compiling unicode-xid v0.2.4
   Compiling itoa v1.0.9
   Compiling rustpython-parser v0.3.0
   Compiling erg_parser v0.6.20
   Compiling serde_repr v0.1.16
   Compiling static_assertions v1.1.0
   Compiling bitflags v1.3.2
   Compiling phf v0.11.2
   Compiling unic-emoji-char v0.9.0
   Compiling unic-ucd-ident v0.9.0
   Compiling itertools v0.10.5
   Compiling embed-doc-image v0.1.4
   Compiling is-macro v0.2.2
   Compiling derive_more v0.99.17
   Compiling lalrpop-util v0.20.0
   Compiling log v0.4.20
   Compiling rustc-hash v1.1.0
   Compiling unicode_names2 v0.6.0
   Compiling rustpython-parser-core v0.3.0
   Compiling url v2.4.1
   Compiling malachite-nz v0.3.2
   Compiling lsp-types v0.93.2
   Compiling malachite-q v0.3.2
   Compiling malachite v0.3.2
   Compiling malachite-bigint v0.1.0
   Compiling rustpython-ast v0.3.0
   Compiling molc v0.1.0
   Compiling els v0.1.32
   Compiling py2erg v0.0.43
   Compiling pylyzer v0.0.43
error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> /home/wil/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pylyzer-0.0.43/src/main.rs:13:31
    |
13  |         let mut lang_server = Server::<PythonAnalyzer, SimplePythonParser>::new(cfg);
    |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^----- an argument of type `Option<Sender<serde_json::value::Value>>` is missing
    |
note: associated function defined here
   --> /home/wil/.cargo/registry/src/index.crates.io-6f17d22bba15001f/els-0.1.32/server.rs:330:12
    |
330 |     pub fn new(cfg: ErgConfig, stdout_redirect: Option<mpsc::Sender<Value>>) -> Self {
    |            ^^^
help: provide the argument
    |
13  |         let mut lang_server = Server::<PythonAnalyzer, SimplePythonParser>::new(cfg, /* Option<Sender<serde_json::value::Value>> */);
    |                                                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Also:
```
❯ cargo --version      
cargo 1.72.0 (103a7ff2e 2023-08-15)
```
and
```
❯ rustup --version
rustup 1.26.0 (2023-05-04)
info: This is the version for the rustup toolchain manager, not the rustc compiler.
info: The currently active `rustc` version is `rustc 1.72.0 (5680fa18f 2023-08-23)`
```

**Comment by 70sh1** [comment]
I have the exact same issue on Windows 11. Installation with `pip` seems to work fine though.

**Comment by wbthomason** [comment]
I should add: I'm using Linux.

**Comment by mtshiba** [comment]
fixed with v0.0.44.

**Comment by wbthomason** [comment]
Confirmed, thanks very much!
